contracts/multisig-prover/src/encoding/abi.rs

```diff
@@ -133,12 +133,11 @@ fn encode_proof(
 pub fn make_operators(worker_set: WorkerSet) -> Operators {
     let mut operators: Vec<(HexBinary, Uint256)> = worker_set
         .signers
-        .iter()
+        .values()
         .map(|signer| {
```

Comment:
```suggestion
        .map(|(weight,pub_key)| {
```
so you can use weight and pub_key instead of signer.0 and signer.1